cypress/e2e/upload-download.cy.ts

```diff
@@ -18,4 +20,13 @@ describe("the user navigates to upload file page should", () => {
         fileName,
     );
   });
+
+  it("download file", () => {
+    const fileName = "sampleFile.jpeg";
+    downloadPage.goToDownloadPage();
+
+    downloadPage.downloadFile();
+
+    cy.verifyDownload(fileName);
```

Comment:
this is ok, but also, it can be done without need of a plugin. `cy.readFile(fileName).should("exist");`

Reply:
Ok, I will keep it in mind! thank you.